interview-questions.md

```diff
@@ -4,57 +4,86 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create....  Read .... U..... Delete 
+  <!--//almost got it :/-->
 
-  Researched answer:
+  Researched answer:CRUD is an acronym that stands for "Create, Read, Update and Delete".
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: put patch and delete
 
-  Researched answer:
+  Researched answer: The primary or most-commonly-used HTTP verbs (or methods, as they are properly called) are POST, GET, PUT, PATCH, and DELETE. These correspond to create, read, update, and delete (or CRUD) operations, respectively.
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: having a parent component passing down to its children?
 
-  Researched answer:
+  Researched answer: class Welcome extends React.Component {
```

Comment:
this question may have been worded a little strange but we are looking for "import, class name extends, render, return, div would be a bonus, export"